dotcom-rendering/src/web/browser/debug/debug.css

```diff
@@ -0,0 +1,43 @@
+/* Layout: Annotate */
+main[data-layout]::before {
```

Comment:
Disadvantage of this route is that the layout would (without additional styling) be annotated at the bottom of the page rather than the top as it is now